feature/home/src/main/java/com/epmedu/animeal/home/presentation/HomeScreenUI.kt

```diff
@@ -72,20 +72,20 @@ internal fun HomeScreenUI(
         }
     }
 
+    LaunchedEffect(key1 = state.timerState) {
```

Comment:
Can you explain why did you create a separate `LaunchedEffect`?

Reply:
Existing launched effect uses key1 = state.timerState, key2 = state.feedingRouteState as keys. We change state of feedingRouteState by stopping it, so the whole process of rejecting expired route repeats and we can see an error as api call for rejecting route gets called second time, when it's already rejected

Reply:
Ah, I see, ok